lucene/analysis/kuromoji/src/test/org/apache/lucene/analysis/ja/dict/UserDictionaryTest.java

```diff
@@ -77,4 +78,19 @@ public void testRead() throws IOException {
     UserDictionary dictionary = TestJapaneseTokenizer.readDict();
     assertNotNull(dictionary);
   }
+
+  @Test(expected = RuntimeException.class)
```

Comment:
Can you use `expectThrows` and check that the message is correct ?

Reply:
I think you mean `assertThrows` that was introduced from JUnit 4.13
https://junit.org/junit4/javadoc/latest/org/junit/Assert.html#assertThrows(java.lang.String,%20java.lang.Class,%20org.junit.function.ThrowingRunnable)

Lucene uses junit 4.12, so we cannot use it (Am I missing something?).

Reply:
Sorry, I missed LuceneTestCase#expectThrows. Will fix this in another patch.

Reply:
Fixed in https://github.com/apache/lucene-solr/pull/830